crates/sui-pg-db/src/temp.rs

```diff
@@ -117,8 +119,11 @@ impl LocalDatabase {
     fn start(&mut self) -> Result<()> {
         if self.process.is_none() {
             self.process = Some(PostgresProcess::start(self.dir.clone(), self.port)?);
-            self.wait_till_ready()
-                .map_err(|e| anyhow!("unable to start postgres: {e:?}"))?;
+            self.wait_till_ready().map_err(|e| {
+                let dir = self.dir.clone().into_os_string().into_string();
+                // may need to add breakpoint/sleep to prevent temp dir from being deleted
+                anyhow!("unable to start postgres: {e:?} check dir {dir:?} for logs")
+            })?;
         }
```

Comment:
- Can we use `with_context` here instead, to preserve error source chaining?
- `Path` has a `display` function that you can use.


```suggestion
            self.wait_till_ready()
                .with_context(|| format!(
                    "Unable to start postgres, check dir for logs", 
                    self.dir.display(),
                ))?;
}
```

Reply:
I added something similar to the above and removed the error message text from `match self.health_check()`.

Reply:
I'm a bit surprised `Display` is needed for this since only `self.dir` needs to be displayed. Curious - which line complains if you remove the `Display` impl block?